### DISCUSSION TITLE
part-ii/2018-1/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#96] [Question comments]
Part II 2018 Algebraic Geometry: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2018-1/

**Comment by Kiolt** [comment]
<details>
<summary>Last part.</summary>

I got $P\in \{Q,(\lambda_1,0),(\lambda_2,0),(\lambda_3,0)\}$.

</details>

**Comment by silkca** [comment]
The first two parts of this question are standard from lectures.

I struggled with the last part but it appears to be just a simple calculation using the geometric interpretation of $\boxplus$ as shown in lectures.

**Reply by artie2000** [reply]
Alternative for last part: $P\boxplus P=Q$ iff $\exists\;f\in L(2P-2Q)$. Use the basis $\{1,x,y\}$ for $L(3Q)$ from (b) to derive the form of $P$.
